src/utils/file-upload-utils.js

```diff
@@ -6,7 +6,7 @@ const { JSDOM } = require("jsdom")
 const { window } = new JSDOM("")
 const DOMPurify = createDOMPurify(window)
 
-const ALLOWED_FILE_EXTENSIONS = [
+export const ALLOWED_FILE_EXTENSIONS = [
```

Comment:
hmm I feel effy about combining es6 and commonjs in the same file, could we standardise this at a file level?

Reply:
sure, fixed!